tests/Unit/TransferFile/Facade/CustomerDirectDebitFacadeTest.php

```diff
@@ -23,6 +23,7 @@
 namespace Digitick\Sepa\Tests\Unit\TransferFile\Facade;
 
 use Digitick\Sepa\PaymentInformation;
+use Digitick\Sepa\TransferFile\Facade\CustomerDirectDebitFacade;
```

Comment:
I don't find any usage for this, can this  be removed ?

Reply:
You are right. I have removed it.